rs/registry/admin/src/main.rs

```diff
@@ -3938,21 +3956,19 @@ async fn main() {
         }
         SubCommand::ProposeToChangeNnsCanister(cmd) => {
```

Comment:
NCR

Can we make the implementation of this just one call instead of inlining? IMHO, in general, we should avoid inlining implementation within a big fan out.

(I think I looked into do that here before, and didn't see a way.)

Reply:
I don't see a good way to do this...